environment_setup/install_clion.sh

```diff
@@ -4,12 +4,56 @@ echo "================================================================"
 echo "Installing CLion"
 echo "================================================================"
 
-# Clion can be installed through snapcraft
-# https://snapcraft.io/clion
-sudo apt-get install snapcraft
-sudo snap install clion --classic
+clion_version="2019.2.2"
+usr_location="/usr/local/bin/clion"
+
+if [ -e ${usr_location} ]
+then
+        echo "================================================================"
+        echo "CLion is already installed"
+        echo "================================================================"
+else
+	# Download clion
+        wget -O ~/Downloads/CLion-${clion_version}.tar.gz "https://download-cf.jetbrains.com/cpp/CLion-${clion_version}.tar.gz"
+
+	# Unzip and symlink to usr location
+        sudo mkdir -p /opt/clion
+        sudo tar xfz ~/Downloads/CLion-${clion_version}.tar.gz -C /opt/clion
+        sudo ln -s /opt/clion/clion-${clion_version}/bin/clion.sh /usr/local/bin/clion
+fi
 
 echo "================================================================"
-echo "Done"
+echo "Installing Bazel Plugin"
 echo "================================================================"
 
+clion_plugin_dir="${HOME}/.CLion2019.2/config/plugins"
+clion_major_version="2019.2"
+bazel_plugin_version="v2019.09.16"
+
+if [ -d "${clion_plugin_dir}/clwb" ]
+then
+	echo "================================================================"
+	echo "Bazel Plugin Already Installed"
+	echo "================================================================"
+else
+	# Download bazel plugin
+	wget -O ~/Downloads/bazelbuild.tar.gz "https://github.com/bazelbuild/intellij/archive/${bazel_plugin_version}.tar.gz"
```

Comment:
similar comment here, we should use `/tmp`

Reply:
done